vcf-dumper/vcf-dumper-lib/pom.xml

```diff
@@ -35,21 +35,6 @@
             <scope>test</scope>
             <version>2.1.0</version>
         </dependency>
-        <dependency>
-            <groupId>org.opencb.cellbase</groupId>
-            <artifactId>cellbase-core</artifactId>
-            <version>${cellbase.version}</version>
-            <exclusions>
-                <exclusion>
-                    <groupId>org.slf4j</groupId>
-                    <artifactId>slf4j-simple</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>org.opencb.biodata</groupId>
-                    <artifactId>biodata-models</artifactId>
-                </exclusion>
-            </exclusions>
-        </dependency>
         <dependency>
             <groupId>com.github.samtools</groupId>
             <artifactId>htsjdk</artifactId>
```

Comment:
We should use a release (instead of snapshot) artifact if possible. The latest version is 2.14.0 so maybe we can try that one and if it causes any problem downgrade to 2.12.0?